internal/service/federatedsettingsidentityprovider/model_federated_settings_identity_provider.go

```diff
@@ -30,6 +29,8 @@ func FlattenFederatedSettingsIdentityProvider(federatedSettingsIdentityProvider
 				"sso_debug_enabled":            federatedSettingsIdentityProvider[i].SsoDebugEnabled,
 				"sso_url":                      federatedSettingsIdentityProvider[i].SsoUrl,
 				"status":                       federatedSettingsIdentityProvider[i].Status,
+				"idp_id":                       federatedSettingsIdentityProvider[i].Id,
```

Comment:
I am slightly confused about the naming of this new attribute. I think it will be good to maintain consistency between the data sources (plural and singular), and ideally with the resource. Right now I am seeing:
Plural data source results
- `okta_idp_id` -> old 20 length id
- `idp_id` -> new 24 length id
- no `id`

Single data source
- `okta_idp_id` -> old 20 length id
- no `idp_id`
- `id` -> new 24 length id


Resource
- `okta_idp_id` -> either 20 or 24 length depending on value used for import
- no `idp_id`
- `id` -> not documented as this stores an encoded value

Please correct me if I am mistaken.

Reply:
You are right! Changed it so single data source and resource will return the `idp_id`. Please take a look again